optuna/integration/cma.py

```diff
@@ -45,16 +45,18 @@ class CmaEsSampler(BaseSampler):
 
         Optimize a simple quadratic function by using :class:`~optuna.integration.CmaEsSampler`.
 
-        .. code::
+        .. testcode:: python
```

Comment:
I think the `python` here is a group name of the doctest feature (c.f. https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testcode). I'm not sure the purpose of the group assignment because I couldn't find the corresponding setup code or cleanup code.
Could you explain why you use the group?

Reply:
Thank you for pointing it out. It's just my mistake that is intended for syntax highlights like `code-block` directive. I fixed it at 8031b5e4.